UPGRADE-2.0.md

```diff
@@ -19,11 +19,14 @@
 - The `curl_ssl_version` option has been removed.
 - The `verify_ssl` option has been removed.
 - The `ca_cert` option has been removed.
+- The `proxy` option has been removed.
```

Comment:
Maybe we should specify here that it got merged in the transport interface?

Reply:
Actually, the option has been removed because it's name/value is dependent on the implementation of the HTTP client, and it was not even used after integrating Httplug. What about this note:

>The `proxy` option has been removed in favour of leaving to the user the burden of configuring the HTTP client options using a custom client.